openlibrary/templates/work_search.html

```diff
@@ -147,16 +147,24 @@ <h1>$_("Search")</h1>
         $# results because of performance issues due to increased load. See
         $# this commit to revert.
         <center>
-            <span class="red">$_("No results found.")</span>
+          <div class="red">No results found.</div>
+          <hr>
+          <div>
             <a href="/search/inside?$urlencode(dict(q=query))">$_('Search for books containing the phrase "%s"?' % query)</a>
+          </div>
+          <hr>
+          <div>
+                <a href="/books/add">Add a new book to Open Library?</a>
+          </div>
         </center>
 
     $elif param and not error and len(docs):
         <div class="search-results-stats">
           $ungettext('1 hit', '%(count)s hits', num_found, count=commify(num_found))
 
           $if num_found > 1:
-            $:render_template("search/sort_options.html", sort)
+            $:render_template("search/sort_options.html", sort)<hr>
+            Can't find what you look for? You can <a href="/books/add">add book to Open Library</href>.
```

Comment:
This might be _too_ prevalent; most users probably will not need to add a book when there are search results.

```suggestion
            $:render_template("search/sort_options.html", sort)
```